### PR TITLE
Legacy database integration

### PR DESCRIPTION
**Pull request by crisingulani**

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
+    type_id_seq = Sequence('product_type_type_id_seq', metadata=Base.metadata)
+
     type_id = Column(
         Integer,
-        primary_key=True,
-        server_default=text(
-            "nextval('product_type_type_id_seq'::regclass)"
-        )
+        type_id_seq,
+        # server_default=type_id_seq.next_value(),
```
Comment (felipelm) [inline models.py]: Keeping this for a future use?

File: requirements.txt [hunk 2]
```diff
@@ -4,4 +4,4 @@ flask-SQLAlchemy==2.3.1
 psycopg2-binary==2.7.5
 graphene[sqlalchemy]
 Flask-GraphQL==2.0.0
-
+sqlacodegen==2.0.0
```
Comment (felipelm) [inline requirements.txt]: Where is this used in the project? Is to generate the models? 

If so please make a wiki showing how to use as a reference.

**Review by felipelm — APPROVED** [review]

File: models.py (hunk 1 above)
Comment (crisingulani) [reply models.py]: Yes

File: requirements.txt (hunk 2 above)
Comment (crisingulani) [reply requirements.txt]: https://github.com/agronholm/sqlacodegen

**Pull request merged**